include/ElasticSearchBase.h

```diff
@@ -34,12 +34,13 @@ class ElasticSearchBase : public TimedRestBatcher, public
 
 protected:
   std::string getElasticSearchUrlonIndex(std::string index);
-  std::string getElasticSearchUrlOnDoc(std::string index, Int64 doc);
-  std::string getElasticSearchUpdateDocUrl(std::string index, Int64 doc);
-  std::string getElasticSearchBulkUrl(std::string index);
+  std::string getElasticSearchUrlOnDoc(std::string index, std::string docType, Int64 doc);
+  std::string getElasticSearchUpdateDocUrl(std::string index, std::string docType, Int64 doc);
+  std::string getElasticSearchBulkUrl(std::string index, std::string docType);
```

Comment:
Why did you add a docType as parameter? in Elasticsearch they removed the document types and right now they keep only _doc for compatibility not used as type but just part of the request url and in newer versions they are going to remove it.

Reply:
yeah i changed one of my doc types...
fixed it now... removed